server/server/initializers/constants.ts

```diff
@@ -1397,6 +1397,10 @@ async function buildLanguages () {
   languages['el'] = 'Greek'
   languages['tok'] = 'Toki Pona'
 
+  // Override Portuguese label
+  languages['pt'] = 'Portuguese (Portugal)'
+  languages['pt-BR'] = 'Portuguese (Brazilian)'
```

Comment:
`pt-BR` is in fact the default locale region for `pt`

> Portuguese (Brazil) [pt_BR] is the default content locale for Portuguese [pt]

https://cldr.unicode.org/translation/translation-guide-general/default-content

Reply:
So I guess we should do something like:
```
  // Override Portuguese label
  // Portuguese (Brazil) [pt_BR] is the default content locale for Portuguese [pt]
  // https://cldr.unicode.org/translation/translation-guide-general/default-content
  languages['pt'] = 'Portuguese (Brazilian)'
  languages['pt-PT'] = 'Portuguese (Portugal)'
```